### PR TITLE
Init mask processing

### PR DESCRIPTION
**Pull request by gluhar2006**

### CODE REVIEW

File: lunavl/sdk/luna_faces.py [hunk 1]
```diff
@@ -163,6 +167,17 @@ def warpQuality(self) -> Quality:
             self._warpQuality = self.estimatorCollection.warpQualityEstimator.estimate(self.warp)
         return self._warpQuality
 
+    @property
+    def mask(self) -> Mask:
+        """
+        Get mask of warped image which corresponding the detection
```
Comment (Zeskbest) [inline lunavl/sdk/luna_faces.py]: Check spelling pls

**Review by Zeskbest — CHANGES_REQUESTED** [review]

File: lunavl/sdk/luna_faces.py (hunk 1 above)
Comment (gluhar2006) [reply lunavl/sdk/luna_faces.py]: fixed

File: lunavl/sdk/errors/errors.py [hunk 2]
```diff
@@ -124,6 +124,7 @@ class LunaVLError:
     HighMemoryUsage = ErrorInfo(110020, "High memory usage", "")
     DetectHumanError = ErrorInfo(110021, "Detect one human body error", "")
     DetectHumansError = ErrorInfo(110022, "Detect humans bodies error", "")
+    EstimationMaskError = ErrorInfo(110023, "Estimation mask error", "")
```
Comment (elenlee) [inline lunavl/sdk/errors/errors.py:127]: `MaskEstimationError`?

File: lunavl/sdk/estimators/face_estimators/mask.py [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+
+        Returns:
+            float in range(0, 1)
+        """
+        return self._coreEstimation.maskInPlace
+
+    @property
+    def isMaskInPlace(self) -> bool:
+        """
+        Get mask is place.
```
Comment (elenlee) [inline lunavl/sdk/estimators/face_estimators/mask.py]: It is not clear

File: lunavl/sdk/estimators/face_estimators/mask.py [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+
+        Returns:
+            float in range(0, 1)
+        """
+        return self._coreEstimation.noMask
+
+    @property
+    def isNoMask(self) -> bool:
+        """
+        Get no mask.
```
Comment (elenlee) [inline lunavl/sdk/estimators/face_estimators/mask.py]: It is not clear

File: lunavl/sdk/estimators/face_estimators/mask.py [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+
+        Args:
+            mask: estimated mask
+        """
+        super().__init__(mask)
+
+    @property
+    def maskInPlace(self) -> float:
+        """
+        Get mask in place probability.
```
Comment (elenlee) [inline lunavl/sdk/estimators/face_estimators/mask.py]: It is not clear. And other comments in the class too

File: lunavl/sdk/estimators/face_estimators/mask.py (hunk 3 above)
Comment (gluhar2006) [reply lunavl/sdk/estimators/face_estimators/mask.py]: updated

File: lunavl/sdk/estimators/face_estimators/mask.py (hunk 4 above)
Comment (gluhar2006) [reply lunavl/sdk/estimators/face_estimators/mask.py]: updated

File: lunavl/sdk/estimators/face_estimators/mask.py (hunk 5 above)
Comment (gluhar2006) [reply lunavl/sdk/estimators/face_estimators/mask.py]: updated

File: lunavl/sdk/errors/errors.py (hunk 2 above)
Comment (gluhar2006) [reply lunavl/sdk/errors/errors.py:127]: see naming for other errors

Comment (elenlee) [reply lunavl/sdk/errors/errors.py:127]: Ok

**Pull request merged**